<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessMainWindow</class>
 <widget class="QMainWindow" name="ProcessMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>947</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1290</width>
    <height>947</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1290</width>
    <height>947</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Template Match</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow,QDialog
{
    
	color: rgb(223, 223, 223);
	background-color: rgb(255, 255, 127);
}
QPushButton:hover
{
color: rgb(43, 43, 43);
	background-color: rgb(255, 213, 41);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1308</width>
      <height>958</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="openPictureButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>70</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{color: rgb(255, 255, 127);
background-color: rgb(255, 85, 0);
border-color: rgb(255, 85, 100);
}
QPushButton:hover
{
color: rgb(43, 43, 43);
	background-color: rgb(255, 213, 41);
}</string>
           </property>
           <property name="text">
            <string>Open
picture</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closePictureButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>70</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
color: rgb(255, 255, 127);
background-color: rgb(70, 70, 70);
border-color: rgb(170, 170, 127);

}
QPushButton:hover
{
color: rgb(43, 43, 43);
background-color: rgb(170, 255, 255);}</string>
           </property>
           <property name="text">
            <string>Close
picture</string>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="logo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Designed by ZHF</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1214</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1280</width>
         <height>20</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>1280</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(138, 138, 138);
background-color: rgb(255, 255, 127);</string>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
color: rgb(255, 255, 127);
background-color: rgb(70, 70, 70);
border-color: rgb(170, 170, 127);

}
QPushButton:hover
{
color: rgb(43, 43, 43);
background-color: rgb(170, 255, 255);}</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="unDoButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
color: rgb(255, 255, 127);
background-color: rgb(70, 70, 70);
border-color: rgb(170, 170, 127);

}
QPushButton:hover
{
color: rgb(43, 43, 43);
background-color: rgb(170, 255, 255);}</string>
           </property>
           <property name="text">
            <string>Undo</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="denoiseButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
color: rgb(255, 255, 127);
background-color: rgb(70, 70, 70);
border-color: rgb(170, 170, 127);

}
QPushButton:hover
{
color: rgb(43, 43, 43);
background-color: rgb(170, 255, 255);}</string>
           </property>
           <property name="text">
            <string>Denoise</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sharpButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>90</horstretch>
             <verstretch>60</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
color: rgb(255, 255, 127);
background-color: rgb(70, 70, 70);
border-color: rgb(170, 170, 127);

}
QPushButton:hover
{
color: rgb(43, 43, 43);
background-color: rgb(170, 255, 255);}</string>
           </property>
           <property name="text">
            <string>Sharp</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="templateMatchButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
color: rgb(255, 255, 127);
background-color: rgb(70, 70, 70);
border-color: rgb(170, 170, 127);

}
QPushButton:hover
{
color: rgb(43, 43, 43);
background-color: rgb(170, 255, 255);}</string>
           </property>
           <property name="text">
            <string>Template
match</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="Line" name="verticalLine">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>750</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>750</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(138, 138, 138);
background-color: rgb(255, 255, 127);</string>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="showPictureWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>630</width>
             <height>750</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>650</width>
             <height>800</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget
{
	background-color: rgb(53, 53, 53);
}</string>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>111</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 255, 255);</string>
            </property>
            <property name="text">
             <string>picture</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="Line" name="verticalLine_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>760</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>760</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(138, 138, 138);
background-color: rgb(255, 255, 127);</string>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="showTemplateWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>371</width>
             <height>750</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>420</width>
             <height>800</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget
{
color: rgb(166, 166, 166);
	background-color: rgb(53, 53, 53);
}</string>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>151</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 255, 255);</string>
            </property>
            <property name="text">
             <string>template</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="Line" name="verticalLine_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>750</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>750</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(138, 138, 138);
background-color: rgb(255, 255, 127);</string>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_open_picture"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_open_picture">
   <property name="text">
    <string>&amp;Open picture</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>&amp;Save as</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>sharpButton</tabstop>
  <tabstop>templateMatchButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
